crates/turborepo-lib/src/commands/info.rs

```diff
@@ -0,0 +1,93 @@
+use std::{env, io};
+
+use sysinfo::{System, SystemExt};
+use thiserror::Error;
+use turborepo_repository::{package_json::PackageJson, package_manager::PackageManager};
+
+use super::CommandBase;
+use crate::{DaemonConnector, DaemonConnectorError};
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("could not get path to turbo binary: {0}")]
+    NoCurrentExe(#[from] io::Error),
+}
+
+pub async fn run(base: CommandBase) -> Result<(), Error> {
+    let system = System::new_all();
+    let connector = DaemonConnector::new(false, false, &base.repo_root);
+    let daemon_status = match connector.connect().await {
+        Ok(_status) => "Running",
+        Err(DaemonConnectorError::NotRunning) => "Not running",
+        Err(_e) => "Error getting status",
+    };
+
+    let package_manager = match PackageJson::load(&base.repo_root.join_component("package.json")) {
+        Ok(package_json) => {
+            match PackageManager::read_or_detect_package_manager(&package_json, &base.repo_root) {
+                Ok(pm) => pm.to_string(),
+                Err(_) => "Not found".to_owned(),
+            }
+        }
+        Err(_) => "Not found".to_owned(),
+    };
```

Comment:
Just some Rust code golf. I'd recommend manually committing this because Rustfmt will probably disagree with this formatting
```suggestion
    let package_manager = PackageJson::load(&base.repo_root.join_component("package.json"))
        .and_then(|package_json|  PackageManager::read_or_detect_package_manager(&package_json, &base.repo_root))
        .map_or_else(|_| "Not found".to_owned(), |pm| pm.to_string());
```